apps/client/src/components/Location/LocationInput.js

```diff
@@ -131,13 +129,18 @@ const LocationInput = ({
       <Paragraph>{text.locationIntro}.</Paragraph>
       <Form onSubmit={handleSubmit(onSubmit)}>
         <LocationFinder
-          setAddress={setAddress}
-          setFocus={setFocus}
-          setErrorMessage={setErrorMessage}
-          postalCode={sessionAddress.postalCode}
+          {...{
+            errors,
+            matomoTrackEvent,
+            setAddress,
+            setErrorMessage,
+            setFocus,
+            topic
+          }}
           houseNumberFull={sessionAddress.houseNumberFull}
           houseNumber={sessionAddress.houseNumberFull}
-          errors={errors}
+          postalCode={sessionAddress.postalCode}
+          address={sessionAddress}
```

Comment:
Looks funny to pass `sessionAddress` AND three of its keys. Why not only sessionAddress and destruct inside `LocationInput.js`. Or at least put a @TODO comment we should refactor this